storage/rocksdb/rdb_datadic.cc

```diff
@@ -2692,6 +2692,20 @@ void Rdb_ddl_manager::erase_index_num(const GL_INDEX_ID &gl_index_id) {
   m_index_num_to_keydef.erase(gl_index_id);
```

Comment:
actually, I was wrong.  There is a lock around this code https://github.com/facebook/mysql-5.6/blob/fb-mysql-5.6.35/storage/rocksdb/rdb_datadic.cc#L3229-L3249